### PR TITLE
docs: ash actions are run in transaction

### PR DESCRIPTION
**Pull request by TwistingTwists**
# Contributor checklist

- [ ] Bug fixes include regression tests
- [ ] Chores
- [x] Documentation changes
- [ ] Features include unit/acceptance tests
- [ ] Refactoring
- [ ] Update dependencies

### CODE REVIEW

File: documentation/topics/actions/actions.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Actions
 
-In Ash, actions are the primary way to interact with your resources. There are five types of actions:
+In Ash, actions are the primary way to interact with your resources. All actions are run in a transaction if the data layer supports it. There are five types of actions:
```
[the remark this answers is not in the document]
Comment (TwistingTwists) [reply documentation/topics/actions/actions.md:3]: This clears one more confusion! 
I'll add that :)

File: documentation/topics/actions/actions.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+In Ash, actions are the primary way to interact with your resources. There are five types of actions:
 
 * [Read](documentation/topics/actions/read-actions.md)
 * [Create](documentation/topics/actions/create-actions.md)
 * [Update](documentation/topics/actions/update-actions.md)
 * [Destroy](documentation/topics/actions/destroy-actions.md)
 * [Generic](documentation/topics/actions/generic-actions.md)
 
-Each action has its own set of options, ways of calling it, and ways of customizing it. See the relevant guide for specifics on each action type. This topic focuses on idiomatic ways to use actions, and concepts that cross all action types.
+Out of above, Create, Update, Destroy actions are run in a transaction if the data layer supports it. Read and Generic actions are NOT run in a transaction. Each action has its own set of options, ways of calling it, and ways of customizing it. See the relevant guide for specifics on each action type. This topic focuses on idiomatic ways to use actions, and concepts that cross all action types.
```
Comment (zachdaniel) [inline documentation/topics/actions/actions.md:11]: Not to keep nitpicking, but they all *can* be run in a transaction. The only thing that is different is the default value for the `transaction?` option. So maybe something like. "All actions can be run in a transaction. Create, update and destroy actions are run in a transaction by default, whereas read and generic actions require opting in with `transaction? true` in the action definition". Maybe that was too wordy, but something like it could make sense.

Comment (TwistingTwists) [reply documentation/topics/actions/actions.md:11]: Clear wording solves problems. 

I'll make those changes.

**Comment by zachdaniel** [comment]
🚀 Thank you for your contribution! 🚀
